scripts/changelog.mjs

```diff
@@ -32,6 +32,10 @@ async function getData() {
         title: "👓 RFC",
         semver: "patch",
       },
+      feat: {
+        title: "🚀 Features",
+        semver: "patch",
+      },
```

Comment:
Maybe we should add the other types from https://github.com/winglang/wing/blob/main/.github/workflows/pull-request-lint.yml while we're at it?

Reply:
The rest are built-in, other than revert which I went ahead and added